cmd/minikube/cmd/config/addons_list.go

```diff
@@ -93,62 +94,94 @@ var stringFromStatus = func(addonStatus bool) string {
 var printAddonsList = func(cc *config.ClusterConfig, printDocs bool) {
 	addonNames := slices.Sorted(maps.Keys(assets.Addons))
 	table := tablewriter.NewWriter(os.Stdout)
+	table.Options(tablewriter.WithHeaderAutoFormat(tw.On))
 
-	table.Options(
-		tablewriter.WithHeaderAutoFormat(tw.On),
-	)
```

Comment:
`table.Options(tablewriter.WithHeaderAutoFormat(tw.On))` replaces this code?

Is this related to the color change? If not please don't include this in this PR. I'm trying to review the color changes and I don't have time to review other changes now.